Exiled.Events/Handlers/Internal/MapGenerated.cs

```diff
@@ -66,7 +66,7 @@ private static void GenerateCache()
         private static void GenerateRooms()
         {
             // Get bulk of rooms with sorted.
-            IEnumerable<GameObject> roomObjects = Object.FindObjectsOfType<RoomIdentifier>().Select(x => x.gameObject).OrderByDescending(x => x.name == "OutSide");
+            IEnumerable<GameObject> roomObjects = Object.FindObjectsOfType<RoomIdentifier>().Select(x => x.gameObject);
```

Comment:
@sanyae2439 Is removing this sorting going to break something? Asking since you added it.

Reply:
Joker it was broken that shit, it was sorting using "OutSide", which is the wrong name ("Outside" is the correct one).
As a result, this line never worked, and I replaced it with `FirstOrDefault` instead, you can see that on Room.cs indeed.

Reply:
Ah, right, ok